aten/src/ATen/native/native_functions.yaml

```diff
@@ -5149,6 +5157,8 @@
 
 - func: zeros(int[] size, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
 
+- func: zeros.SymInt(SymInt[] size, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
```

Comment:
totally not relevant to this PR but quick question - at some point, do we wanna consider using codegen to automatically generate / enforce that all ops taking in concrete ints have a symint alternative? Maybe the infra isn't in a complete enough state that it's worth pursuing that yet?

Reply:
I think it's pretty good idea, but we are still pretty early to pursue this yet :(